CHANGELOG

```diff
@@ -1,7 +1,8 @@
 [v#.#.#] ([month] [YYYY])
   - Font: Improve language support
+  - Ruby: Upgrade ruby version to 3.4.4
   - Upgraded gems:
-    - thor
+    - rails, thor
```

Comment:
This is a bit inconsistent, particularly considering the major Rails bump. Not sure what we've done before, but I'd expect "upgraded gems" to be mostly minor stuff.